packages/dd-trace/src/opentracing/tracer.js

```diff
@@ -42,6 +42,7 @@ class DatadogTracer extends Tracer {
     this._url = this._exporter._url
     this._sampler = new Sampler(config.sampleRate)
     this._peers = config.experimental.peers
+    this._enableGetRumData = config.experimental.enableGetRumData
```

Comment:
The feature is still experimental right now so this cannot be added for the moment.

Reply:
@rochdev can we not use it to at least try it out? I understand it's a beta/experimental feature.

I needed to save the config somewhere, the initial config object is not saved anywhere.

The current implementation is completely bugged and the tests don't actually work, it's a false positive on the test sweet. They perform expectations in a callback without using `done()`.

This is what happens today due to how the method is attempting to read the config: https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/proxy.js#L128

```
TypeError: Cannot read property 'experimental' of undefined
```

We need to store this config value somewhere or it cannot be accessed.

Reply:
Yes, sorry I misread this as being in `config.js`. It's perfectly fine to set it on the tracer like this.